src/modules/esl-related-target/core/esl-related-target.ts

```diff
@@ -19,17 +21,21 @@ export class ESLRelatedTarget extends ESLMixinElement {
 
   /** Selector/query ({@link ESLTraversingQuery}) to find related ESLToggleable element */
   @attr({name: ESLRelatedTarget.is}) public selector: string;
+  /** Action to synchronize between toggleables */
+  @attr({name: ESLRelatedTarget.is + '-action', defaultValue: 'all'}) public action: ESLRelatedTargetActionType;
 
-  /** @returns related toggleable instance */
-  public get $relatedTarget(): ESLToggleable | null {
+  /** @returns related toggleable instances */
+  public get $targets(): ESLToggleable[] {
     const {selector} = this;
-    if (!selector) return null;
-    const target = ESLTraversingQuery.first(selector);
-    return target instanceof ESLToggleable ? target : null;
+    if (!selector) return [];
+    const targets = ESLTraversingQuery.all(selector);
```

Comment:
Incorrect usage of Traversing Query base element is missing to support relative pathes